packages/backend/src/apps/delay/actions/delay-until/index.ts

```diff
@@ -30,27 +30,31 @@ const action: IRawAction = {
 
   async run($) {
     const defaultTime = '00:00'
+    // trim the date and time for user
     const { delayUntil, delayUntilTime } = $.step.parameters
+    const delayUntilString = (delayUntil as string).trim()
```

Comment:
consider converting to string instead of casting

Reply:
thanks for the catch!